app/models/photo.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const mongoose = require('mongoose')
+
+const photoSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true
+  },
+  imageUrl: {
+    type: String,
+    required: true
+  },
+  tags: {
+    type: Array,
+    required: false
+  }
+}, {
+  timestamps: true
+})
+
+module.exports = mongoose.model('Photo', photoSchema)
```

Comment:
I'm not sure why this file is showing in your PR as if there weren't an `app/models/photo.js` file in development (which there definitely is), but it looks like that's the source of the merge conflict.

Before merging routes, let's just make sure that feature branch has rebased off of the most recent development